components/DualSmallViews.js

```diff
@@ -21,70 +21,89 @@ import Header from './Header'
 import { linkCheck } from './functionHelpers'
 
 export default class DualSmallViews extends Component {
+  constructor(props) {
```

Comment:
For constructors that don't do anything with the props, a syntactic shortcut available in React Native is to leave out the explicit constructor and just say

```suggestion
    state = {
      aspectRatio: 1,
      aspectRatioBottom: 1
    }
```